server/test/services/ti/TrustedIntermediaryServiceTest.java

```diff
@@ -362,91 +365,266 @@ private void setupTIAccount(
+        addCSRFToken(
+            fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "clientFirst",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "ClientLast",
+                        "dob",
+                        "2022-07-07",
+                        "emailAddress",
+                        "email2123",
+                        "tiNote",
+                        "unitTest",
+                        "phoneNumber",
+                        "0000000000")));
+    Form<EditTiClientInfoForm> form =
+        formFactory.form(EditTiClientInfoForm.class).bindFromRequest(requestBuilder.build());
+    Form<EditTiClientInfoForm> returnForm = service.updateClientInfo(form, tiGroup, testAccount.id);
+    assertThat(returnForm.error("phoneNumber").get().message())
+        .isEqualTo("This phone number is not valid");
   }
 
   @Test
-  public void updateApplicantDateOfBirth_ApplicantDobUpdated() throws ApplicantNotFoundException {
-    setupTIAccount("First", "2021-11-11", "fake@email.com", tiGroup);
+  public void editTiClientInfo_PhoneNumberNotDigitsValidationFail()
+      throws ApplicantNotFoundException {
     Http.RequestBuilder requestBuilder =
-        addCSRFToken(fakeRequest().bodyForm(ImmutableMap.of("dob", "2021-09-09")));
-    Form<UpdateApplicantDobForm> form =
-        formFactory.form(UpdateApplicantDobForm.class).bindFromRequest(requestBuilder.build());
-    AccountModel account = repo.lookupAccountByEmail("fake@email.com").get();
-    Form<UpdateApplicantDobForm> returnedForm =
-        service.updateApplicantDateOfBirth(tiGroup, account.id, form);
-    assertThat(returnedForm.hasErrors()).isFalse();
-    assertThat(account.newestApplicant().get().getApplicantData().getDateOfBirth().get().toString())
-        .isEqualTo("2021-09-09");
+        addCSRFToken(
+            fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "clientFirst",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "ClientLast",
+                        "dob",
+                        "2022-07-07",
+                        "emailAddress",
+                        "email2123",
+                        "tiNote",
+                        "unitTest",
+                        "phoneNumber",
+                        "0000000er0")));
+    Form<EditTiClientInfoForm> form =
+        formFactory.form(EditTiClientInfoForm.class).bindFromRequest(requestBuilder.build());
+    Form<EditTiClientInfoForm> returnForm = service.updateClientInfo(form, tiGroup, testAccount.id);
+    assertThat(returnForm.error("phoneNumber").get().message())
+        .isEqualTo("A phone number must contain 10 digits");
```

Comment:
question (blocking): I think this error message is not quite right; should it be something like "A phone number must only contain digits" instead?